### PR TITLE
#622: Add ECHO command

### LINKED ISSUE #622: Add support for ECHO command.
DiceDB currently doesn't support the [ECHO](https://redis.io/docs/latest/commands/echo/) command. I would like to add that.

### PR DESCRIPTION
**Pull request by hgupta12**
Resolves #622 : Add support for ECHO command.

## Description
- This  PR adds support for [ECHO](https://redis.io/docs/latest/commands/echo/) command to DiceDB.
- Added unit tests (internal/eval/eval_test.go)  
- Added integration tests (integration_tests/commands/echo_test.go)  

## Screenshots
image

### CODE REVIEW

File: internal/eval/commands.go [hunk 1]
```diff
@@ -25,6 +25,13 @@ var (
 		Eval:  evalPING,
 		Arity: -1,
 	}
+	echoCmdMeta = DiceCmdMeta{
+		Name:     "ECHO",
+		Info:     `ECHO returns the string given as argument.`,
+		Eval:     evalECHO,
+		Arity:    1,
+		KeySpecs: KeySpecs{BeginIndex: 1},
```
Comment (apoorvyadav1111) [inline internal/eval/commands.go:33]: Hi, for commands with arguments without keys, we can omit KeySpecs.

File: internal/eval/eval.go [hunk 2]
```diff
@@ -72,6 +72,16 @@ func evalPING(args []string, store *dstore.Store) []byte {
...
 }
 
+// evalECHO returns the argument passed by the user 
+func evalECHO(args []string, store *dstore.Store) []byte {
+	if len(args) != 1 {
+		return diceerrors.NewErrArity("ECHO")
+	}
+
+	b := clientio.Encode(args[0], false)
+	return b
```
Comment (apoorvyadav1111) [inline internal/eval/eval.go:82]: nit:
```suggestion
	return clientio.Encode(args[0], false)
```

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]

**Comment by apoorvyadav1111** [comment]
Hi, thanks for adding this feature. I left some minor comments.

**Comment by hgupta12** [comment]
@apoorvyadav1111 I have resolved the comments.

**Review by apoorvyadav1111 — APPROVED** [review]

**Review by lucifercr07 — APPROVED** [review]

**Pull request merged**